timing/mkAllTests

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
No, this is not run by travis. All the files in 'timing' are for running timing analysis on the BES.